### PR TITLE
ECS cli target

### PR DESCRIPTION
**Pull request by kaihendry**

### CODE REVIEW

File: AWS-docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 version: '2'
 services:
     bugzilla:
-        image: uneet/bugzilla-customisation:$COMMIT
-        restart: on-failure
```
Comment (franck-boullier) [inline AWS-docker-compose.yml:5]: @kaihendry why do we need to change this behavior?

File: deploy.sh [hunk 2]
```diff
@@ -8,12 +8,9 @@ STAGE=dev
 show_help() {
 cat << EOF
 Usage: ${0##*/} [-p]
-
 By default, deploy to dev environment on AWS account 812644853088
```
Comment (franck-boullier) [inline deploy.sh]: @kaihendry This is misleading since the AWS account number can be different.

File: deploy.sh [hunk 3]
```diff
@@ -8,12 +8,9 @@ STAGE=dev
...
-
 	-p          PRODUCTION 192458993663
 	-d          DEMO 915001051872
```
Comment (franck-boullier) [inline deploy.sh]: @kaihendry This is misleading since the AWS account number can be different.

File: deploy.sh [hunk 4]
```diff
@@ -8,12 +8,9 @@ STAGE=dev
 show_help() {
 cat << EOF
 Usage: ${0##*/} [-p]
-
 By default, deploy to dev environment on AWS account 812644853088
-
 	-p          PRODUCTION 192458993663
```
Comment (franck-boullier) [inline deploy.sh]: @kaihendry This is misleading since the AWS account number can be different.

File: env-setup.bash [hunk 5]
```diff
@@ -14,12 +14,12 @@ cat << EOF > .env
 API_ACCESS_TOKEN=$(ssm API_ACCESS_TOKEN)
 MYSQL_HOST=db
 MYSQL_PORT=3306
-MYSQL_ROOT_PASSWORD=$(ssm MYSQL_ROOT_PASSWORD)
+MYSQL_ROOT_PASSWORD=$(ssm MYSQL_ROOT_PASSWORD_INS)
```
Comment (franck-boullier) [inline env-setup.bash:17]: @kaihendry this looks incorrect to me.

File: env-setup.bash [hunk 6]
```diff
@@ -14,12 +14,12 @@ cat << EOF > .env
...
 MYSQL_DATABASE=bugzilla
 MYSQL_USER=bugzilla
-MYSQL_PASSWORD=$(ssm MYSQL_PASSWORD)
-PARAMS_URL=https://raw.githubusercontent.com/unee-t/bugzilla-customisation/master/params/local-params.json
-SES_SMTP_PASSWORD=$(ssm SES_SMTP_PASSWORD)
-SES_SMTP_USERNAME=$(ssm SES_SMTP_USERNAME)
-SES_VERIFIED_SENDER=dev.case@unee-t.com
+MYSQL_PASSWORD=$(ssm MYSQL_PASSWORD_INS)
```
Comment (franck-boullier) [inline env-setup.bash:20]: @kaihendry this looks incorrect to me.

File: env-setup.bash [hunk 7]
```diff
@@ -14,12 +14,12 @@ cat << EOF > .env
...
+PARAMS_URL=https://raw.githubusercontent.com/unee-t-ins/bugzilla-customisation/master/params/local-params.json
+SES_SMTP_PASSWORD=$(ssm SES_SMTP_PASSWORD_INS)
```
Comment (franck-boullier) [inline env-setup.bash:22]: @kaihendry this looks incorrect to me.

File: env-setup.bash [hunk 8]
```diff
@@ -14,12 +14,12 @@ cat << EOF > .env
...
+SES_SMTP_USERNAME=$(ssm SES_SMTP_USERNAME_INS)
```
Comment (franck-boullier) [inline env-setup.bash:23]: @kaihendry this looks incorrect to me.

**Review by franck-boullier — CHANGES_REQUESTED** [review]

File: AWS-docker-compose.yml (hunk 1 above)
Comment (kaihendry) [reply AWS-docker-compose.yml:5]: ecs-cli doesn't understand the restart keyword, so removed it

**Pull request closed without merging**